src/main.zig

```diff
@@ -497,6 +497,8 @@ const usage_build_generic =
     \\  -search_static_only            Only search for static libs.
     \\  -T[script], --script [script]  Use a custom linker script
     \\  --version-script [path]        Provide a version .map file
+    \\  --undefined-version            Allow the linker to link with symbols with undefined version
+    \\  --no-undefined-version         Cause a fatal error if the linker attempts to link to a symbol with an undefined version
```

Comment:
Have a look at `lld`'s description which is more accurate as versioning only comes into play from version scripts.

https://github.com/llvm/llvm-project/blob/dc1dc60ea8786d4a2462d157842501d76593f9e7/lld/ELF/Options.td#L480-L482